### PR TITLE
Fix the omnomcom wrapped endpoint

### PR DESCRIPTION
**Pull request by ysbrandB**
Fix the endpoints used by omnomcom wrapped. See also https://github.com/saproto/OmNomCom-Wrapped/pull/27.

### CODE REVIEW

File: routes/api.php [hunk 1]
```diff
@@ -13,7 +14,8 @@
     });
     /* Routes related to the User API */
     Route::group(['prefix' => 'user', 'as' => 'user::'], static function () {
-        Route::group(['middleware' => ['auth:api'], 'as' => 'qr::'], static function () {
+        Route::get('info', fn () => Auth::user())->name('info')->middleware('auth:api');
```
Comment (JonaMata) [inline routes/api.php:17]: Why is this here?

**Review by JonaMata — CHANGES_REQUESTED** [review]
LAGTM

File: routes/api.php (hunk 1 above)
Comment (ysbrandB) [reply routes/api.php:17]: This was first in the userapi controller, but that has been removed. It is used to check if the login was succesfull.

**Review by JonaMata — APPROVED** [review]
LGTM
